src/envoy/token/iam_token_info.cc

```diff
@@ -82,12 +85,16 @@ Envoy::Http::MessagePtr IamTokenInfo::prepareRequest(
     insertStrListToProto(body, kScopesField, scopes_, "");
   }
 
-  if (!delegates_.empty() || !scopes_.empty()) {
-    std::string bodyStr =
-        MessageUtil::getJsonStringFromMessage(body, false, false);
-    message->body() =
-        std::make_unique<Buffer::OwnedImpl>(bodyStr.data(), bodyStr.size());
-  }
+
+  Envoy::ProtobufWkt::Value val;
+  vals.set_bool_value(true);
+  (*body.mutable_struct_value()->mutable_fields())[kIncludeEmail].Swap(&val);
+
+  std::string bodyStr =
+      MessageUtil::getJsonStringFromMessage(body, false, false);
+  message->body() =
+      std::make_unique<Buffer::OwnedImpl>(bodyStr.data(), bodyStr.size());
```

Comment:
I did not see any changes in iam_token_info_test.  It should have unit-test to test the generated request body